pkg/vm/ch.go

```diff
@@ -24,43 +23,6 @@ const (
 	cloudConsoleBin = "cloud-console"
 )
 
-// startCloudConsole Starts the cloud console for the vm on it's private network ip
-func (m *Machine) startCloudConsole(ctx context.Context, namespace string, networkAddr net.IPNet, machineIP net.IPNet, ptyPath string, logs string) (string, error) {
```

Comment:
I think the cloud console can stay since the user network will always have a mycelium IP. the console can still work over the mycelium IP in case the VM is not reachable for any reason. 
- So the ip of the vm is the VM private ip (private range)
- The console listen port is calculated the same way
- The console IP itself is the mycelium IP assigned to this user mycelium instance.